<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMessageBox</class>
 <widget class="QDialog" name="MyMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMessageBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widgetTitleBar" native="true">
       <property name="styleSheet">
        <string notr="true">#widgetTitleBar{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(103, 171, 236, 255), stop:0.2 rgba(95, 163, 228, 255),stop:0.4 rgba(82, 147, 211, 255),stop:0.6 rgba(61, 125, 189, 255),stop:0.8 rgba(50, 111, 174, 255),stop:1 rgba(35, 95, 158, 255));
}

QToolButton {
	border-image: url(:/resources/close.png);
}

QToolButton:hover {
	border-image:url(:/resources/close-hover.png);
}

QToolButton:pressed {
	border-image:url(:/resources/close-pressed.png);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelTitle">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonClose">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetCentral" native="true">
       <property name="styleSheet">
        <string notr="true">#widgetCentral {
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labelIcon">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/resources/information.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="topMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QLabel" name="labelTips">
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetBottom" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widgetBottom {
background-color: rgba(238, 238, 238,255);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>168</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButtonOK">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButtonCancel">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
